### ISSUE TITLE
normalize_intensities filter should also normalize losses

### ISSUE THREAD

**Issue by florian-huber** [matchms/matchms#69]
Currently the filter only normalized peak intensities, but that would give very different results for running
```python
spectrums = [add_losses(s) for s in spectrums]
spectrums = [normalize_intensities(s) for s in spectrums]
```
than for 
```python
spectrums = [normalize_intensities(s) for s in spectrums]
spectrums = [add_losses(s) for s in spectrums]
```

**Comment by florian-huber** [comment]
--> PR #70

**Comment by florian-huber** [comment]
This solved as part of #70 .
